src/components/DocumentationTopic/DocumentationNav/VersionToggle.vue

```diff
@@ -0,0 +1,283 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <NavMenuItemBase class="nav-menu-setting version-container">
+    <div :class="{ 'version-toggle-container': manyVersions }">
+      <!-- Faux element to get width of select, with current element-->
+      <select
+        class="version-dropdown version-sizer"
+        ref="version-sizer"
+        aria-hidden="true"
+        tabindex="-1"
+      >
+        <option selected>{{versionModel}}</option>
+      </select>
+      <!-- Faux element is above -->
+      <label
+        :for="versionList ? 'version-toggle' : null"
+        class="nav-menu-setting-label"
+      >Version:</label>
+      <select
+        v-if="versionList.length >1 "
```

Comment:
```suggestion
        v-if="versionList.length"
```

Reply:
@hqhhuang it actually need it to be greater than 1 because if it’s 1 then it should go into the singleVersionPage property